File: platforms/swift/examples/scene/material/normal_texture.swift
import FragmentColor

let renderer = Renderer()
let normal_map_bytes = [
    128, 128, 255, 255,
    128,   128, 255, 255,
    128,   128, 255, 255,
    128,   128, 255, 255,
]
let normal_map = try await renderer.createTexture((normal_map_bytes, [2, 2]))
let mat = Material.pbr().normalTexture(normal_map).normalScale(1.2)

File: platforms/python/examples/scene/material/normal_texture.py
from fragmentcolor import Material, Renderer

renderer = Renderer()
normal_map_bytes = [
    128, 128, 255, 255,
    128,   128, 255, 255,
    128,   128, 255, 255,
    128,   128, 255, 255,
]
normal_map = renderer.create_texture(normal_map_bytes, size=[2, 2])
mat = Material.pbr().normal_texture(normal_map).normal_scale(1.2)